builds/besd/Dockerfile

```diff
@@ -48,12 +48,17 @@ MAINTAINER support@opendap.org
 USER root
 
 # Update and install the needful.
-RUN set -e \
-    && dnf install -y which bc python3 emacs vim  \
-    && dnf install -y procps \
-    && dnf update -y \
-    && dnf clean all \
-    && python3 -m pip install awscli
+RUN set -e && dnf install -y python3.11
```

Comment:
One thing about separate 'RUN' command, each one makes a new layer in the Docker filesystem, and that makes for a larger and slower container. The pattern of chaining them together with '&&' is recommended on the docker site. (I thought this was the case and just checked since it's one of those 'optimizations' that's easy to get wrong).

Reply:
And it makes it completely impossible to identify what failed in the **&&** chain **&&** of **&&** things.. We can slam them back together once we are convinced it's working.